Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -322,22 +310,123 @@ public void Exit()
             _exitTriggered = true;
         }
 
+        private Order GetOrderById(int id)
+        {
+            Order order = null;
+
+            if (_orders.TryGetValue(id, out order) == false)
+            {
+                var clientOrder = _algorithm.Transactions.GetOrderById(id);
+
+                if (order != null)
```

Comment:
Ok, that's great. What should I do? I've only resently started using git.
Should I merge my branch with master?

On Thu, Jul 2, 2015 at 4:52 PM, Michael <notifications@github.com> wrote:

> In Engine/TransactionHandlers/BrokerageTransactionHandler.cs
> <https://github.com/QuantConnect/Lean/pull/134#discussion_r33821631>:
>
> > @@ -322,22 +310,123 @@ public void Exit()
> >              _exitTriggered = true;
> >          }
> >
> > +        private Order GetOrderById(int id)
> > +        {
> > +            Order order = null;
> > +
> > +            if (_orders.TryGetValue(id, out order) == false)
> > +            {
> > +                var clientOrder = _algorithm.Transactions.GetOrderById(id);
> > +
> > +                if (order != null)
>
> See my recent change where I moved the storage of the order queue and
> orders to the transaction handler: 9c1b0f4
> <https://github.com/QuantConnect/Lean/commit/9c1b0f4616e599f77e33f30c3c1ee8f6b4ef3c74>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/QuantConnect/Lean/pull/134/files#r33821631>.
>